openlibrary/plugins/openlibrary/js/availability.js

```diff
@@ -231,15 +231,15 @@ function initAvailability() {
                                 if (work.status === 'open' || work.status === 'borrow_available') {
                                     $(cta).append(`<a href="/books/${work.openlibrary_edition}/x/borrow" ` +
                                                   'class="cta-btn cta-btn--available" ' +
-                                                  `data-ol-link-track="${work.status}">${
+                                                  `data-ol-link-track="CTAClick|Borrow">${
```

Comment:
Blocker: This needs to either be `CTAClick|Borrow` or `CTAClick|Read` based on the `${work.status}

Reply:
```suggestion
                                                  `data-ol-link-track="CTAClick|${work.status === 'open' ? 'Read' : ' Borrow'}">${
```